### DISCUSSION TITLE
[Flag]Duplicate 'Serbia' but different code

### DISCUSSION THREAD

**Discussion by teacher-zhou** [fomantic/Fomantic-UI#1649] [Elements (button, image, label, loader, ...)] [answered]
I find that we have the same name of 'Serbia' but with different code 'cs','rs', is that correct?

**Comment by TheJltres** [comment, marked as answer]
Hi, yes that is correct.
RS is only for Serbia: https://www.iso.org/obp/ui/#iso:code:3166:RS

CS is for Serbia and Montenegro: https://www.iso.org/obp/ui/#iso:code:3166:CS
CS has no ISO flag as you can see heare: https://en.wikipedia.org/wiki/ISO_3166-1

Now Montenegro is ME

Should cs be changed or removed?

**Comment by lubber-de** [comment]
In #1190 we already removed `cs` as it does not have a separate ISO flag as @TheJltres pointed out and thus isn't supported by Tweemoji.
However, for backward compatibility we cannot remove 'cs' in the current branch.
